From 906922ee321d64e2ce8458147e67d4892696fb58 Mon Sep 17 00:00:00 2001
From: Valery Ushakov <valery.ushakov@bell-sw.com>
Date: Wed, 22 May 2024 17:07:38 +0300
Subject: [PATCH 075/157] Makefile: fix make -s detection

Only check the first word of MAKEFLAGS for 's', that's where all the
single letter options are collected.

MAKEFLAGS contains _all_ make flags, so if any command line argument
contains a letter 's', the silent test will be false positive.  Think
e.g. make 'DESTDIR=.../aports/main/mdadm/pkg/mdadm' install
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index adac7905..446710bd 100644
--- a/Makefile
+++ b/Makefile
@@ -157,7 +157,7 @@ ifndef UDEVDIR
  UDEVDIR = /lib/udev
 endif
 
-ifeq (,$(findstring s,$(MAKEFLAGS)))
+ifeq (,$(findstring s,$(firstword -$(MAKEFLAGS))))
 	ECHO=echo
 else
 	ECHO=:
-- 
2.41.0

